setup.py

```diff
@@ -1,53 +1,27 @@
+#!/usr/bin/env python
+
 # this script allows us to publish sureal in PyPi and the use it using 'pip install sureal'
 # run 'python setup.py sdist bdist_wheel' to create the distribution files within sureal/python/src/
 
-import os
-import setuptools
-from numpy.distutils.core import setup
-
-PACKAGE_NAME = 'sureal'
-
-parent_dir = os.path.dirname(__file__)
-
-
-def _version():
-    """ Get the local package version.
-    """
-    path = os.path.join(PACKAGE_NAME, "version.py")
-    namespace = {}
-    with open(path) as stream:
-        exec(stream.read(), namespace)
-    return namespace["__version__"]
-
-
-with open(os.path.join(parent_dir, "README.md"), "r") as fh:
-    long_description = fh.read()
+from setuptools import setup
 
-try:
-    import pypandoc
-    long_description = pypandoc.convert_text(long_description, 'rst', format='md')
-except ImportError:
-    print("pypandoc module not found, could not convert Markdown to RST")
 
 setup(
-    name=PACKAGE_NAME,
-    version=_version(),
+    name="sureal",
+    setup_requires=["setupmeta"],
+    versioning="dev",
     author="Zhi Li",
     author_email="zli@netflix.com",
-    description="Subjective quality scores recovery from noisy measurements.",
-    long_description=long_description,
-    packages=setuptools.find_packages(),
-    install_requires=['numpy>=1.12.0', 'scipy>=0.17.1', 'matplotlib>=2.0.0', 'pandas>=0.19.2'],
```

Comment:
All of these are automatic with setupmeta, one can verify they indeed are auto-filled correctly and look good via `python setup.py explain`